tools/accuracy_checker/configs/yolo-v3-tf.yml

```diff
@@ -0,0 +1,125 @@
+models:
+  - name: yolo-v3-tf
+
+    launchers:
+      - framework: tf
+        model:   public/yolo-v3-tf/yolo-v3.pb
+        inputs:
+          - name: 'input_1'
+            type: INPUT
+            shape: 1,416,416,3
+        output_names:
+          - conv2d_58/BiasAdd
+          - conv2d_66/BiasAdd
+          - conv2d_74/BiasAdd
+        adapter:
+            type: yolo_v3
+            anchors: "10,13,  16,30,  33,23,  30,61,  62,45,  59,119,  116,90,  156,198,  373,326"
+            #num: 3
+            num: 9
+            coords: 4
+            classes: 80
+            anchor_masks: [[6,7,8],[3,4,5],[0,1,2]]
+            raw_output: True
+            outputs:
+              - conv2d_58/BiasAdd
+              - conv2d_66/BiasAdd
+              - conv2d_74/BiasAdd
+
+    datasets:
+      - name: ms_coco_detection_80_class_without_backgound
+
+        preprocessing:
+          - type: bgr_to_rgb
+          - type: normalization
+            std: 255.0
+          - type: resize
+            size: 416
+
+        postprocessing:
+          - type: resize_prediction_boxes
+          - type: filter
+            apply_to: prediction
+            min_confidence: 0.001
+            remove_filtered: True
+          - type: nms
+            overlap: 0.5
+          - type: clip_boxes
+            apply_to: prediction
+
+        metrics:
+          - type: map
+            integral: 11point
+            ignore_difficult: true
+            presenter: print_scalar
+          - type: coco_precision
+            max_detections: 100
+            threshold: 0.5
+
+  - name: yolo-v3-tf
+
+    launchers:
+      - framework: dlsdk
+        tags:
+          - FP32
+        model:   public/yolo-v3-tf/FP32/yolo-v3-tf.xml
+        weights: public/yolo-v3-tf/FP32/yolo-v3-tf.bin
+        adapter:
+            type: yolo_v3
+            anchors: "10,13,  16,30,  33,23,  30,61,  62,45,  59,119,  116,90,  156,198,  373,326"
+            #num: 3
```

Comment:
please remove excess comments in configs

Reply:
done